data/json/item_groups.json

```diff
@@ -2429,7 +2428,6 @@
       ["gin", 20],
       ["rum", 14],
       ["tequila", 12],
-      ["chem_ethanol", 10],
```

Comment:
Pure ethanol is intended in food alcohol groups. It represents spirits like Everclear.